avogadro/qtplugins/apbs/apbs.cpp

```diff
@@ -4,6 +4,7 @@
 ******************************************************************************/
 
 #include "apbs.h"
+
```

Comment:
???

Reply:
Not sure why there's an extra blank line here.

Reply:
I added it to separate the header for the source from the local module includes following the recommended include ordering:

1. header with definitions
2. local includes
3. wider project includes
4. 3rd party libraries includes
5. standard library includes

This cuts down on include bloat because you include the most common ones last ensuring you can make the most of the already included libraries from the more niche files you would have to include either way.

Reply:
@ghutchis its just for readability and easy distinction